api/bin/chainflip-cli/src/main.rs

```diff
@@ -266,6 +269,50 @@ async fn request_redemption(
 	Ok(())
 }
 
+async fn request_internal_transfer(
```

Comment:
This has a lot in common with `request_redemption`, seems like there should be plenty of scope to share code.